tests/test_SklearnGradientBoostingConverters.py

```diff
@@ -0,0 +1,113 @@
+# -------------------------------------------------------------------------
```

Comment:
Why have you added this file? We already have https://github.com/onnx/sklearn-onnx/blob/master/tests/test_sklearn_gradient_boosting_converters.py, with the unit tests.

Reply:
Sorry wrong rebase.